packages/clay-layout/src/Row.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+type TNumberOfColumns = 1 | 2 | 3 | 4 | 5 | 6;
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Vertical positioning of row contents
+	 */
+	align?: 'start' | 'center' | 'end';
```

Comment:
`align-items` has more than these 3 options. [MDN on align-items](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items)

Reply:
Yeah I am following bootstraps docs on their alignment classes. https://getbootstrap.com/docs/4.0/layout/grid/#vertical-alignment

I think we can start with these 3 for now and then evaluate later if people need more.